README.md

```diff
@@ -195,6 +195,13 @@ By default mode, it would ask you to enter the master password
 
 > ciphertool -Dchange  (in UNIX)
 
+4. Change the Port by modifying the conf/axis2.xml file
```

Comment:
Do you mean the carbon.xml?

Even so, manually changed the configuration will be overridden with the configurations in the deployment.toml.

Correct way to change the port numbers are mentioned in the product documentation https://is.docs.wso2.com/en/latest/references/default-ports/#change-the-offset-for-default-ports